extensions/PhabBugz/lib/WebService.pm

```diff
@@ -291,6 +299,92 @@ sub obsolete_attachments {
     return { result => \@updated_attach_ids };
 }
 
+sub needs_review {
+    my ($self, $params) = @_;
+    ThrowUserError('phabricator_not_enabled')
+        unless Bugzilla->params->{phabricator_enabled};
+    my $user = Bugzilla->login(LOGIN_REQUIRED);
+    my $dbh  = Bugzilla->dbh;
+
+    my $reviews = get_needs_review();
+
+    # map author phids to bugzilla users
+    my $author_id_map = get_phab_bmo_ids({
+        phids => [
+            uniq
+            grep { defined }
+            map { $_->{fields}{authorPHID} }
+            @{$reviews}
```

Comment:
nit: curlies not required for de-referencing an array here.
perlcritic is set to allow this, with [-References::ProhibitDoubleSigils].

(in a lot of cases I'll prefer `$foo = $some->{data}{structure}; @$foo`
to `@{ $some->{data}{structure} }` )

Reply:
will do – all my personal stuff has `References::ProhibitDoubleSigils` enabled, so it's habitual